src/server/api/lib/utils.js

```diff
@@ -15,3 +15,7 @@ export function mapFieldsToModel(fields, model) {
   })
   return resolvers
 }
+
+export function capitalize(word) {
+  return word[0].toUpperCase() + word.slice(1)
```

Comment:
this will fail on empty fields, so you'll need to guard against that.

Reply:
++